portfolio/src/main/webapp/script.js

```diff
@@ -14,6 +14,17 @@
 $(document).ready(function() {
   getComments();
 
+  $('.deleter').click(function(event) {
```

Comment:
nit: it's preferable to use arrow functions: https://google.github.io/styleguide/jsguide.html#features-functions-arrow-functions

Reply:
done!